modules/BTBidAdapter.js

```diff
@@ -0,0 +1,184 @@
+import { config } from '../src/config.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { deepSetValue, isPlainObject, logWarn } from '../src/utils.js';
+import { BANNER } from '../src/mediaTypes.js';
+import { ortbConverter } from '../libraries/ortbConverter/converter.js';
+
+const BIDDER_CODE = 'blockthrough';
+const GVLID = 815;
+// reach out for approval to marketing@blockthrough.com
+const ENDPOINT_URL = 'https://pbs.sample.com';
```

Comment:
what is this fake value?

Reply:
We want to provide real endpoint values only after publisher approval. Are they required?

Reply:
am i missing the line where you overwrite this value with the value in configuration?

Reply:
@patmmccann no we left it out intentionally. I'll talk to the team about putting in the real endpoint.

Reply:
I must be missing something, is your intention for the publisher to fork the project and change this constant?

Reply:
@PavloMalashnyak please update the endpoint with the real one. Thanks